src/main/java/uk/gov/hmcts/cmc/claimstore/documents/content/FullDefenceResponseContentProvider.java

```diff
@@ -70,6 +78,10 @@
         content.put("evidences", evidences);
         content.put("evidenceComment", evidenceComment);
         content.put("formNumber", DEFENCE_FORM_NO);
+
+        fullDefenceResponse.getDirectionsQuestionnaire().ifPresent(dq ->
+            content.put("hearingContent", hearingContentProvider.mapDirectionQuestionnaire.apply(dq)));
```

Comment:
`mapDirectionQuestionnaire.apply` in tests is fine but in code a functional method should be used as the assignment target for a lambda expression or method reference.